frontend/src/containers/pages/PlayScenarioPage/PlayScenarioPage.jsx

```diff
@@ -83,7 +79,13 @@ export default function PlayScenarioPage() {
     console.log("reset");
     setPrevious(null);
     history.replace(`/play/${scenarioId}/singleplayer`);
-  };
+  }, [sceneId]);
```

Comment:
missing dependencies, for `useCallback` you **always** want to recompute on dependency changes

why? 

say `scenarioId` changes (i.e the user navigates to the different param), you will still have the memoized version of the `reset` function that points to the old `scenarioId`

Reply:
idk the linter should handle this for you but the rule is off